frontend/src/Setting/Admin/ItemAdd/ItemAddPane.tsx

```diff
@@ -54,31 +54,28 @@ function ItemAddPane() {
   });
   const toast = useToast();
 
-  async function onSubmit(data: IItemAddFormInput) {
-    addItem(data);
+  async function onSubmit(data: TItemAddFormInput) {
+    try {
+      await addItem(data);
+      showToast("アイテムを追加しました", "success");
+    } catch (e) {
+      showToast("アイテムの追加に失敗しました", "error");
+    }
     reset();
```

Comment:
`reset()`は成功時のみでいいのでは
失敗ならデータを変えて再試行するのが自然
値段とかのデータはクリアしてほしくないはず
```suggestion
    try {
      await addItem(data);
      reset();
      showToast("アイテムを追加しました", "success");
    } catch (e) {
      showToast("アイテムの追加に失敗しました", "error");
    }
```